export/exporter.go

```diff
@@ -120,7 +120,7 @@ func (e *exporter) export(info protocol.ToolInfo) error {
 		}
 
 		if len(f.defRangeIDs) > 0 || len(f.useRangeIDs) > 0 {
-			_, err = e.emitContains(f.docID, append(f.defRangeIDs, f.useRangeIDs...))
+			_, err = e.emitContains(f.docID, deduplicate(append(f.defRangeIDs, f.useRangeIDs...)))
```

Comment:
It would be better if we have a comment about why duplications would happen here.

Reply:
Will do